.travis.yml

```diff
@@ -103,8 +103,8 @@ before_install:
     - echo "Python site version = ${PYSITE}"
     # Install python dependencies. NOTE:  if you change this list, also
     # change them in the dependency tarball script.
-    - pip install -q numpy scipy matplotlib cmake cython astropy ephem healpy numba
```

Comment:
This is fine- can you also add toml to the toast/travis/travis_ubuntu_14.04.sh file?

Reply:
https://github.com/zonca/toast/commit/e3cd60c6d7c668616bcde35426660e6f3e1b86c0